data/users.yaml

```diff
@@ -28,10 +28,6 @@ consumers:
     description: Customers can use SPIFFE IDs to establish [mTLS](https://en.wikipedia.org/wiki/Mutual_authentication#mTLS) connections between Ghostunnel proxies with built-in support for obtaining X.509-SVID identities via the [SPIFFE Workload API](/docs/latest/spiffe-about/spiffe-concepts/#spiffe-workload-api)
     link: https://github.com/square/ghostunnel
     logo: square.jpg
-  - name: VMware Secrets Manager
```

Comment:
I am legitimately confused how this one happened - this PR was supposed to just be a re-do of PR #299 after the commits failed to sign. I'll undo it.

Reply:
Hehe, no worries. Thanks for reverting the change, but the build is failing as it can't parse this file anymore. It looks like the indentation is off.

Reply:
Tried one more time - should be good now :)